src/NuGetPackageVerifier/Program.cs

```diff
@@ -25,6 +25,13 @@ public static int Main(string[] args)
 
             // TODO: Get this from the command line
             var ignoreAssistanceMode = IgnoreAssistanceMode.ShowAll;
+            var hideInfoLogs = false;
+
+            if (args.Length > 0 && string.Equals("--quiet", args[0], StringComparison.OrdinalIgnoreCase))
```

Comment:
Ideally all of this goes away if \ when we make this an MSBuild task (which is why I said I wasn't super vested in this). If you think this works, we'll live with it 👍